### DISCUSSION TITLE
node/npm version in postUpgradeTasks

### DISCUSSION THREAD

**Discussion by Hackwar** [renovatebot/renovate#17994] [Request Help] [answered]
Some dependencies require us to update manifest files in our codebase, which we prepared a script for. Our dependencies and our script require nodejs 16+ and package-lock.json v2, but I can't get it to use the right version. It is always stuck on nodejs 14 and package-lock.json v2. I tried `constraints` and `force: {constraints: {}}` and none work. We are selfhosting and the right nodejs version is loaded via docker sidecar. What am I missing?

**Comment by rarkins** [comment, marked as answer]
Sounds like #8111

**Reply by Hackwar** [reply]
Indeed. So I'm going to subscribe to that issue and see what happens.
